src/test/java/br/unb/cic/analysis/samples/DFPBaseSample.java

```diff
@@ -6,15 +6,19 @@
 public class DFPBaseSample {
     private String text;
 
+    public DFPBaseSample(String text){
+        this.text = text;
+    }
+
     public void cleanText(){
-        DFPBaseSample inst = new DFPBaseSample();
+        DFPBaseSample inst = new DFPBaseSample("the  the dog");
         inst.normalizeWhiteSpace(); //Left
         inst.removeComments();
         inst.removeDuplicateWords(); //Right
```

Comment:
Acho que aqui precisaria de uma linha a mais com `this.text = inst.text`, ou então usar o this em todas, já que está manipulando um objeto novo

Reply:
mais que isso, @nathaliafab , faz sentido criar o objeto aí? na versão que você encaminhou não tinha isso, até onde lembro. @galilasmb não copiasse o código de @nathaliafab ?

Reply:
Com o this não funciona, temos que saber qual objeto está manipulando, por isso usamos o objeto instanciado em vez do this.

Reply:
como assim o this não funciona, @galilasmb ? deveria sim ser possível analisar esse método com this. se não é possível, há algo a ser ajustado na implementação da análise.

Reply:
Posso testar mais, mas na última vez ele não conseguia achar o points-to.

Reply:
@pauloborba @galilasmb usando spark realmente o this não funciona. Precisa de um new. Usando o cha, funcionou aqui para mim.

Em um exemplo em que temos this.chamadaDeMetodo() esse método não é acessado pela análise que usa spark. Pelo menos em OA, funciona assim atualmente.